qml/CountryPage.qml

```diff
@@ -106,10 +107,11 @@ BasePage {
             }
         }
 
-        service.call("luna://com.palm.systemservice/setPreferences", JSON.stringify(request), setPreferencesSuccess, setPreferencesFailure);
+        service.call("luna://com.palm.systemservice/setPreferences", JSON.stringify(request), setPreferencesSuccess(request), setPreferencesFailure);
```

Comment:
You sure about "(request)" ? it should be a function argument, not a function call here.